inst/stan/epinowcast.stan

```diff
@@ -22,78 +22,128 @@ data {
   array[s] int csl; // cumulative version of the above
   array[s] int sg; // how snapshots are related
   int dmax; // maximum possible report date
+
   // Observations
   array[s, dmax] int obs; // obs for each primary date (row) and report date (column)
   array[n] int flat_obs; // obs stored as a flat vector
   array[t, g] int latest_obs; // latest obs for each snapshot group
+
+  // Expectation model
+  array[2] real eobs_lsd_p; // Standard deviation for expected final observations
+
   // Reference day model
-  int refp_fnrow; // how many unique pmfs there are
-  array[s] int refp_findex; // how each date links to a pmf
-  int refp_fncol; // number of effects to apply
-  matrix[refp_fnrow, refp_fncol + 1] refp_fdesign; // design matrix for pmfs
-  int refp_rncol; // number of standard deviations to use for pooling
-  matrix[refp_fncol, refp_rncol + 1] refp_rdesign; // Pooling pmf design matrix 
-  int model_refp; // parametric distribution (0 = none, 1 = exp, 2 = lognormal, 3 = gamma, 4 = loglogistic)
+  int model_refp;
+  int refp_fnrow;
+  array[s] int refp_findex;
+  int refp_fncol;
+  matrix[refp_fnrow, refp_fncol + 1] refp_fdesign;
+  int refp_rncol;
+  matrix[refp_fncol, refp_rncol + 1] refp_rdesign; 
+  array[2] real refp_mean_int_p;
+  array[2] real refp_sd_int_p;
+  array[2] real refp_mean_beta_sd_p;
+  array[2] real refp_sd_beta_sd_p; 
```

Comment:
So I felt like it was repeating known information as these parts come from a function with documented output and it was the same every time. The idea of removing was to make the stan code easier/clearer to read just because there was less of it. Perhaps a halfway house is documenting what each thing is once and then saying it is the same for all other prefixes

Reply:
I understand the rationale